Mathlib/Geometry/Manifold/VectorField/LieBracket.lean

```diff
@@ -342,6 +342,38 @@ lemma mlieBracket_add_left
   simp only [← mlieBracketWithin_univ] at hV hV₁ ⊢
   exact mlieBracketWithin_add_left hV hV₁ (uniqueMDiffWithinAt_univ _)
 
+omit [IsManifold I 2 M] [CompleteSpace E] in
+/-- We have `[0, W] = 0` for all vector fields `W`: this depends on the junk value 0
```

Comment:
Could you move this earlier in the file, where `[IsManifold I 2 M] [CompleteSpace E]` are not already introduced?

Reply:
Done!